product-web-app/tests/authService.test.ts

```diff
@@ -1,16 +1,27 @@
-import { expect } from 'chai';
+import chai, { expect } from 'chai';
+import * as sinon from 'sinon';
+import sinonChai from 'sinon-chai';
```

Comment:
To use them, you will need to install an additional package:
 npm install ts-sinon sinon-chai @types/sinon @types/sinon-chai --save-dev